ci/prs.yml

```
@@ -141,6 +149,7 @@ jobs:
     - collect_build_data
     - check_for_release
     - check_changelog_entry
+    - check_compatibility_protobuf
```

```suggestion
    - compatibility_head_protobuf
```

Azure will not start the build if the name here doesn't match the job name.